appveyor.yml

```diff
@@ -0,0 +1,23 @@
+version: 2.0.{build}
+branches:
+  only:
+  - master
+image: Visual Studio 2017
+configuration: Release
+before_build:
+  appveyor-retry nuget restore
+build:
+  parallel: true
+  verbosity: minimal
+test:
+  assemblies:
+    - MetadataExtractor.Tests.dll
+test_script:
+  - dotnet test .\MetadataExtractor.Tests\MetadataExtractor.Tests.csproj --configuration Release --no-build
```

Comment:
Tests could target `net451` as well. Ideally we'd test both build configurations.